Contents/Sketch/colors.js

```diff
@@ -7,7 +7,9 @@ var applyColors = function (newColors, docData) {
     color.alpha = newColors[i].Opacity;
     colors.push(color);
   }
-  colors = MSArray.dataArrayWithArray(colors);
-  appController.globalAssets().setPrimitiveColors(colors);
-  appController.globalAssets().objectDidChange();
+  [assetColors addColors:colors];
+
+  // The following line is throwing the error: is not a function
+  // appController.globalAssets().objectDidChange();
+  
 }
```

Comment:
@nolastan objectDidChange() line is throwing an error so I re-commented it out to finish up the other changes. My first guess is the method is deprecated. I'm taking a break on this repo for now. Can you look into a fix if you want the line back in there?